fluffy/static/js/home.js

```diff
@@ -27,12 +27,14 @@ $(document).ready(function() {
             fh.slideUp(duration);
             pb.slideDown(duration, function() {
                 transitioningModes = false;
+                $(document).off("paste", pastefile);
```

Comment:
the page can start in paste mode (add `?text` to the end of the URL), don't think it will do the right thing in that case